pex/vendor/__main__.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import print_function
+
+import os
+import subprocess
+import sys
+
+from . import vendor_specs
+from ..common import safe_delete, safe_rmtree
+
+
+class VendorizeError(Exception):
+  """Indicates an error was encountered updating vendored libraries."""
+
+
+def vendorize(vendor_spec):
+  cmd = ['pip', 'install', '--upgrade', '--no-compile', '--target', vendor_spec.target_dir,
```

Comment:
Is it necessary to delete the old copy before doing this, or does the upgrade do the right thing? Would it be safer to delete and then always do a fresh install?

Reply:
Upgrade does the right thing and ~the world relies on this day in, day out.